_layouts/navbar.md

```diff
@@ -1,2 +1,4 @@
 * [Home Page](/)
-* [⭐ Star The Repo](https://github.com/open-sauced/intro)
\ No newline at end of file
+* [⭐ Star The Repo](https://github.com/open-sauced/intro)
+* [:us:](/)
+* [:fr:](/translations/fr/)
```

Comment:
If the sidebar has been implemented in #42, do we need really these line here? 
I don't think so, it would raise merge conflicts. 
I would rather wait until #42 gets merged and then go ahead with this PR.